src/App.tsx

```diff
@@ -54,6 +54,22 @@ const theme = createTheme({
       },
     },
   },
+  components: {
+    MuiInputLabel: {
+      styleOverrides: {
+        shrink: {
+          "@media (max-width: 600px)": {
+            color: "#F2F0EB !important",
+          },
+          "@media (min-width: 600px)": {
+            fontSize: "#712B75 !important",
```

Comment:
What's the necessity of using `!important` here?

Reply:
I tried to explain my decision in the comment above. BTW fontSize couldn't have value "#712B75 !important". My bad.